libs/sdk-ui-ext/src/internal/components/pluggableVisualizations/testHelpers.ts

```diff
@@ -0,0 +1,44 @@
+// (C) 2020 GoodData Corporation
+import { IImplicitDrillDown } from "../../interfaces/Visualization";
+import { IDrillEventIntersectionElement, IDrillEvent, VisType } from "@gooddata/sdk-ui";
+import { IAttribute, IInsight, IInsightDefinition, uriRef } from "@gooddata/sdk-model";
+export function createDrillDefinition(fromAttribute: IAttribute, targetUri: string): IImplicitDrillDown {
+    return {
+        implicitDrillDown: {
+            from: {
+                drillFromAttribute: { localIdentifier: fromAttribute.attribute.localIdentifier },
+            },
+            target: {
+                drillToAttribute: {
+                    attributeDisplayForm: uriRef(targetUri),
+                },
+            },
+        },
+    };
+}
+
+export function wrapUriIdentifier(
```

Comment:
This name is confusing, how about `insightDefinitionToInsight`?

Reply:
Good point :-) Renamed